### PR TITLE
Bug null apps table

### PR DESCRIPTION
**Pull request by amyleblond**
Changes:

**ranking_status.html**: adding front end loop if statement for if apps table is null

**views.py** : adding pull from course table if apps table is empty, adding variable to use on front end when apps table is empty

Note - tried renaming both the existing queryset and a new one -- because of the merge "course__" cannot be used for rename-- but is used on front end. Hence no rename and if loop on front end. 

Testing:

Empty Apps table:
screen shot 2018-02-12 at 7 30 43 pm

Entries in App Table:
screen shot 2018-02-12 at 7 31 35 pm

### CODE REVIEW

File: taworks/taform/templates/taform/ranking_status.html [hunk 1]
```diff
@@ -19,7 +19,19 @@ <h4>Below is a listing of the status of graduate student rankings for the upcomi
 	<th class="tg-8fvv">Instructor Email</th>
 	<th class="tg-8fvv">Ranking Status</th>
 	</tr>
-{% for c in ranking_status %}
+{% if emptyApps %}
+	{% for c in ranking_status %}
+<tr>
+	<td class="tg-yw4l"><a target="_blank" href="https://team4.uwaterloo.ca/taform/instructor/{{c.url_hash}}/">MSCI&nbsp;{{c.course_id}}</a></td>
+	<td>{{c.section}}</td>
+	<td><b>No Applicants</b></td>
```
Comment (j2kan) [inline taworks/taform/templates/taform/ranking_status.html]: i think this should be 0 to be consistent

**Review by j2kan — DISMISSED** [review]
lgtm just 1 comment about naming it 0.

**Comment by amyleblond** [comment]
good call!

**Comment by smwatts** [comment]
When you click "send out ranking links" the view goes from:

screen shot 2018-02-12 at 8 52 34 pm

To:
screen shot 2018-02-12 at 8 51 52 pm

(1) Do we want to continue to display the table? 
(2) To continue to display the home/logout button we have to pass 'AC' : AC into the dictionary when rendering the template.

**Comment by amyleblond** [comment]
@smwatts good catch. ill uupdate

**Comment by amyleblond** [comment]
screen shot 2018-02-12 at 9 14 09 pm

on my last merge i kept two posts for button click - first one was not the most up to date hence the issue. should be fine now &&& home and logout

**Review by smwatts — APPROVED** [review]
LGTM, awesome job! @j2kan do we want to include the timestamp of last sent for emails? Or does it not really matter since he does it once a term? (irrelevant to this PR lol - just a random thought!)

**Comment by smwatts** [comment]
@amyleblond the header/buttons with the new CSS aren't shown in your screenshots. Have you run collect static? Just want to make sure there isn't an issue with displaying the CSS

**Comment by amyleblond** [comment]
yes sorry I did run collect static -- sometimes my chrome doesnt show css  on local but does on server. This is safari: 
screen shot 2018-02-12 at 10 02 09 pm

Also I think a time stamp could be good here too! Just kind of as a reminder -- would also identify if it's been too long without responses? Talk about it wednesday

**Comment by j2kan** [comment]
I agree with having the timestamp for the email as well! good reminder and feedback.

**Pull request merged**